src/home/homework-directory.c

```diff
@@ -290,7 +290,7 @@ int home_resize_directory(
         if (r < 0)
                 return r;
 
-        r = home_store_embedded_identity(new_home, setup->root_fd, h->uid, embedded_home);
+        r = home_store_embedded_identity(new_home, setup->root_fd, embedded_home);
```

Comment:
please split this commit out, we should just merge this right-away

Reply:
well that's the behavior I observed. I even printed out the offending flags and O_NOFOLLOW and compared against the definitions of the various O_* flags, and O_NOFOLLOW was the one that was set and caused the check to fail. Removing O_NOFOLLOW in AccountsService fixed the error I was getting from homed too.

No idea if it's intentional :shrug: